### PR TITLE
Add task name to the README for each model

### PR DESCRIPTION
**Pull request by WanliZhong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,23 +14,23 @@ Guidelines:
...
-| [CRNN-CN](./models/text_recognition_crnn)             | 100x32   | 73.52  | 322.16  | 239.76 | 166.79 | ---   |
-| [PP-ResNet](./models/image_classification_ppresnet)   | 224x224  | 56.05  | 602.58  | 98.64  | 75.45  | ---   |
-| [MobileNet-V1](./models/image_classification_mobilenet)| 224x224 | 9.04   | 92.25   | 33.18  | 145.66\* | ---   |
-| [MobileNet-V2](./models/image_classification_mobilenet)| 224x224 | 8.86   | 74.03   | 31.92  | 146.31\* | ---   |
-| [PP-HumanSeg](./models/human_segmentation_pphumanseg) | 192x192  | 19.92  | 105.32  | 67.97  | 74.77  | ---   |
-| [WeChatQRCode](./models/qrcode_wechatqrcode)          | 100x100  | 7.04   | 37.68   | ---    | ---    | ---   |
-| [DaSiamRPN](./models/object_tracking_dasiamrpn)       | 1280x720 | 36.15  | 705.48  | 76.82  | ---    | ---   |
-| [YoutuReID](./models/person_reid_youtureid)           | 128x256  | 35.81  | 521.98  | 90.07  | 44.61  | ---   |
-| [MPPalmDet](./models/palm_detection_mediapipe)        | 256x256  | 15.57  | 89.41   | 50.64  | 145.56\* | ---   |
+| Model | Topic | Input Size | INTEL-CPU (ms) | RPI-CPU (ms) | JETSON-GPU (ms) | KV3-NPU (ms) | D1-CPU (ms) |
```
Comment (fengyuentau) [inline README.md:17]: Topic -> Task

File: README.md [hunk 2]
```diff
@@ -14,23 +14,23 @@ Guidelines:
...
+|-------|-------|------------|----------------|--------------|-----------------|--------------|-------------|
+| [YuNet](./models/face_detection_yunet)                | Face Detection               | 160x120  | 1.45   | 6.22    | 12.18  | 4.04   | 86.69 |
+| [SFace](./models/face_recognition_sface)              | Face Recognition             | 112x112  | 8.65   | 99.20   | 24.88  | 46.25  | ---   |
+| [LPD-YuNet](./models/license_plate_detection_yunet/)  | License Plate Detection      | 320x240  | ---    | 168.03  | 56.12  | 154.20\* |       |
+| [DB-IC15](./models/text_detection_db)                 | Text detection               | 640x480  | 142.91 | 2835.91 | 208.41 | ---    | ---   |
```
Comment (fengyuentau) [inline README.md:22]: Text detection ->  Text Detection

File: README.md [hunk 3]
```diff
@@ -14,23 +14,23 @@ Guidelines:
...
+| [DB-TD500](./models/text_detection_db)                | Text detection               | 640x480  | 142.91 | 2841.71 | 210.51 | ---    | ---   |
```
Comment (fengyuentau) [inline README.md:23]: Text detection ->  Text Detection

File: README.md [hunk 4]
```diff
@@ -14,23 +14,23 @@ Guidelines:
...
+| [CRNN-EN](./models/text_recognition_crnn)             | Text detection               | 100x32   | 50.21  | 234.32  | 196.15 | 125.30 | ---   |
```
Comment (fengyuentau) [inline README.md:24]: Text detection -> Text Recognition

File: README.md [hunk 5]
```diff
@@ -14,23 +14,23 @@ Guidelines:
...
+| [CRNN-CN](./models/text_recognition_crnn)             | Text detection               | 100x32   | 73.52  | 322.16  | 239.76 | 166.79 | ---   |
```
Comment (fengyuentau) [inline README.md:25]: Text detection ->  Text Recognition

**Review by fengyuentau — APPROVED** [review]
Thank you! Please take a look at the comments below.

Others look good to me 👍

File: README.md [hunk 6]
```diff
@@ -14,23 +14,23 @@ Guidelines:
...
+| [PP-ResNet](./models/image_classification_ppresnet)   | Image Recognition            | 224x224  | 56.05  | 602.58  | 98.64  | 75.45  | ---   |
+| [MobileNet-V1](./models/image_classification_mobilenet)| Image Classification         | 224x224 | 9.04   | 92.25   | 33.18  | 145.66\* | ---   |
+| [MobileNet-V2](./models/image_classification_mobilenet)| Image Classification         | 224x224 | 8.86   | 74.03   | 31.92  | 146.31\* | ---   |
+| [PP-HumanSeg](./models/human_segmentation_pphumanseg) | Human Segmentation           | 192x192  | 19.92  | 105.32  | 67.97  | 74.77  | ---   |
+| [WeChatQRCode](./models/qrcode_wechatqrcode)          | QR Code Detection and Parsing | 100x100  | 7.04   | 37.68   | ---    | ---    | ---   |
```
Comment (ShiqiYu) [inline README.md:30]: parsing or decoding?

Comment (WanliZhong) [reply README.md]: > parsing or decoding?

In the [documentation](https://github.com/opencv/opencv_contrib/tree/master/modules/wechat_qrcode) of WeChat QR code, they use `parsing`.

**Comment by fengyuentau** [comment]
Merge as is

**Pull request merged**